### ISSUE TITLE
Additional core icons

### ISSUE THREAD

**Issue by M0nk3yy** [foundryvtt/foundryvtt#8038]
### User Experience

I would like to request some additional core icons. I was originally going to submit these to dnd5e, but dnd5e now exclusively uses core icons.

I bought the REXARD icon megapack and have some icons that I would particularly like to see adopted (so I can submit them to dnd5e for use in the SRD). The "ModernToolsIcons" and "RpgWeaponIcons" packs in particular have several icons that aren't currently in Foundry.

Below is a selection of those I would personally like to see, with the default names of the icons in their packs, and copies of the images.

Crowbar: ModernToolsIcons - modern_28_b.png
Trident: RpgWeaponsIcons_bg - nw_b_10.png
Lance: RpgWeaponsIcons_bg/addons - 04_b.PNG
Morningstar: RpgWeaponsIcons_bg/addons - 140_b.PNG
Crossbow: RpgWeaponsIcons_bg/addons - 144_b.PNG

![modern_28_b](https://user-images.githubusercontent.com/66137312/187971201-83ce03ce-8eab-4be7-9d78-657d0b20aeba.png)
![nw_b_10](https://user-images.githubusercontent.com/66137312/187971367-345a08cf-8580-4396-9617-f0e50ea6e951.PNG)
![04_b](https://user-images.githubusercontent.com/66137312/187971396-de39d5b5-7870-428a-9742-8cea768d960c.PNG)
![140_b](https://user-images.githubusercontent.com/66137312/187971413-9f4e1c64-ef6a-4cc1-aba7-7b7c31f02ebc.PNG)
![144_b](https://user-images.githubusercontent.com/66137312/187971485-77f48652-3dc9-4817-aebf-0dde22a09c87.PNG)

The RpgWeaponIcons "addons" folder contains several other useful ones (cutlasses, more crossbows, glaives etc.). Hopefully these icons are all also part of the licensed pack that Foundry has.

**Comment by aaclayton** [comment]
I don't believe these particular packs were included in the HumbleBundle pack that we purchased, but we can certainly buy more icon packs if there are specific ones that would round out gaps in our offerings. Please feel free to post some suggested additioonal REXARD packs and where we can buy them.

**Comment by xdy** [comment]
These would help with those playing sci-fi (and to some extent modern) games:
https://graphicriver.net/item/scifi-skill-icon-pack/19618342
https://graphicriver.net/item/scifi-icons/19763763
https://graphicriver.net/item/scifi-armor-icons/19843015

And, of course, this one fills out fantasy nicely: https://assetstore.unity.com/packages/2d/gui/icons/fairytale-icons-megapack-125520 (Unless it's already included? Not sure. Think I recognize some of the icons. But then, Rexard is everywhere...)

**Comment by M0nk3yy** [comment]
I have a license so I'll lend them to you ;) Unfortunate that we seem to have different contents of bundles.

But yes, xdy's links are accurate although I normal browse via unity store:

https://assetstore.unity.com/packages/2d/gui/icons/modern-tools-icons-51370
https://assetstore.unity.com/packages/2d/gui/icons/rpg-weapons-icons-48546

There is some downside that many packs (including in the bundle I bought) often have icons in common with other packs. But these packs do contain useful icons that aren't currently available. 

@xdy do note that that first sci-fi bundle you linked, definitely does have many icons that Foundry currently includes, but they are listed under sub areas such as "magic" rather than anything connotated with sci-fi.

**Comment by xdy** [comment]
I thought I had seen some of those icons. But, well, Rexard is everywhere. :)

**Comment by p4535992** [comment]
+1

**Comment by T-Whiz** [comment]
Here are some suggestions for rounding out gaps where I couldn't find something equivalent in the current set of icons, along with potential categories:

[Modern Tools Icons (Rexard) ](https://assetstore.unity.com/packages/2d/gui/icons/modern-tools-icons-51370)
tools-hand: Crowbar (modern_28_b.png)

[Prehistoric Age Icons (Rexard)](https://assetstore.unity.com/packages/2d/gui/icons/prehistoric-age-icons-104432)
weapons-slings: Sling (PrehistoricIcon_54_b.PNG)

[Technology and Skill Icons (Yusuf Artun)](https://assetstore.unity.com/packages/2d/gui/icons/technology-and-skill-icons-69115)
containers-misc: Cart (Cart.png)
environment-settlement: Tent (Logistics.png, Advanced_Logistics.png)
tools-hand: Abacus (Mathematics.png)
tools-hand: Painter's palette (Art.png)

[RPG Craft Resources (A-ravlik)](https://assetstore.unity.com/packages/2d/gui/icons/rpg-craft-resources-224629)
commodities-biological: Finger: Preview image 21 (row 10)
commodities-biological: Snout: Preview image 25 (row 5)
commodities-bones: Antler (moose): Preview image 38 (row 4)
commodities-materials: Hay bale: Preview image 29,(row 6)
commodities-tech: Drill parts: Preview image 32 (row 3)
commodities-treasure: Needle (magic): Preview image 7 (row 9)
commodities-treasure: Painting/artwork: Preview image 8 (row 11)
commodities-wood: Furniture: Preview image 28 (row 6), Preview image 33 (row 5)
commodities-wood: Sled: Preview image 33 (row 6), Preview image 38 (row 4)
consumables-food: Cookies/biscuits: Preview image 19 (rows 7, 8)
consumables-food: Sandwich: Preview image 38 (row 1)
containers-misc: Cart: Preview image 28 (row 6)
containers-misc: Potion pack: Preview image 7 (row 12)
containers-misc: Wheelbarrow: Preview image 29 (row 6)
environment-creatures: Preview image 29 (rows 3, 4), Preview image 31 (row 7)
equipment-chest: Dress: Preview image 15 (rows 6, 10)
equipment-head: Eyepatch: Preview image 8 (row 10)
equipment-head: Hats/headwear: Preview image 16 (rows 9, 10), Preview image 36 (row 6), Preview image 39 (row 5), Preview image 40 (row 10)
tools-cooking: Rolling pin: Preview image 8 (row 2)
tools-cooking: Spoon (metal): Preview image 8 (row 11), Preview image 15 (row 3)
tools-cooking: Spoon (wooden): Preview image 8 (rows 5, 6), Preview image 31 (row 3)
tools-instruments: Preview image 7 (row 11), Preview image 8 (rows 3, 7, 9, 11), Preview image 23 (rows 3, 5, 7, 8, 10), Preview image 33 (row 7), Preview image 35 (row 5), Preview image 38 (row 2)
tools-laboratory: Funnel: Preview image 7 (row 8)
tools-laboratory: Still/lab equipment: Preview image 8 (rows 5, 11)
tools-hand: Carpenters Tools: Preview image 7 (row 1)
tools-hand: Clothing iron: Preview image 8 (row 3)
tools-hand: Thimble: Preview image 7 (row 8)
weapons-misc: Blowgun: Preview image 29 (row 1)
weapons-misc: Whip: Preview image 8 (row 6)

**Comment by M0nk3yy** [comment]
True icons for crowbars, blowguns, whips, and slings? Be still my beating heart!

**Comment by aaclayton** [comment]
A significant blocker here is that the Unity Store asset license is not compatible with our use case. The assets we obtained via Humble Bundle were offered under a different (more open) license.

**Comment by T-Whiz** [comment]
> A significant blocker here is that the Unity Store asset license is not compatible with our use case. The assets we obtained via Humble Bundle were offered under a different (more open) license.

Would something like this work (Extended License option)? https://graphicriver.net/item/modern-tools-icons/19768225?s_rank=112

**Comment by aaclayton** [comment]
No, there are some problematic terms in that license also.

**Comment by T-Whiz** [comment]
Thanks for taking a look. Maybe in the future an opportunity will come up to obtain something in the appropriate style with a more open license.

**Comment by T-Whiz** [comment]
This appears to have a license much closer to the one from the Humble Bundle: https://www.gamedevmarket.net/terms-conditions#pro-licence

https://www.gamedevmarket.net/asset/prehistoric-age-icons
https://www.gamedevmarket.net/asset/modern-tools-icons-8699
https://www.gamedevmarket.net/asset/medieval-technology-and-skill-icons-5397
